RELEASE-NOTES.md

```diff
@@ -1,4 +1,18 @@
 ## RELEASE NOTES
+### Version 3.0.0-epsilon
```

Comment:
Will these all eventually end up being simply `Version 3.0.0` without all the pre-release versions mentioned?

Reply:
Yes, that's correct. All of these Greek-alphabet suffix ones are just pre-releases, which will move to `3.0.0` once QA approves everything for release to Production.